drivers/mspi/mspi_dw.h

```diff
@@ -27,6 +27,7 @@
 #define CTRLR0_FRF_SSP		1UL
 #define CTRLR0_FRF_MICROWIRE	2UL
 #define CTRLR0_DFS_MASK		GENMASK(4, 0)
+#define CTRLR0_DFS32_MASK	GENMASK(20, 16)
```

Comment:
This field should be added and used conditionally, based on `SSI_VERSION_2`, as it is not present e.g. in revision 1.03a -bits 19:16 are used for CFS (Control Frame Size, for Microware) field and bits 21:20 are reserved there.

Reply:
Fixed to define the mask as 0x0 when SSI_VERSION_2 is not defined- does this work?